src/main/java/org/jcodec/common/io/ByteBufferChannel.java

```diff
@@ -0,0 +1,84 @@
+package org.jcodec.common.io;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+
+import org.jcodec.common.io.NIOUtils;
+import org.jcodec.common.io.SeekableByteChannel;
+
+/**
+ * This class is part of JCodec ( www.jcodec.org ) This software is distributed
+ * under FreeBSD License
+ * 
+ * Implements a seekable byte channel that wraps a byte buffer
+ * 
+ * @author The JCodec project
+ * 
+ */
+public class ByteBufferChannel implements SeekableByteChannel {
```

Comment:
they work in slightly different way
one is basically for writing another one for reading
i guess i should add testcases